go.mod

```diff
@@ -1,8 +1,9 @@
-module github.com/abraverm/openengine
+module openengine
 
 go 1.14
 
 require (
+	github.com/abraverm/openengine v0.0.0-20201027075718-848cd7d650eb
```

Comment:
what is the point of changing "github.com/abraverm/openengine" to "openengine" when you need to add this line with a specific commit as a requirement?

Reply:
This was automatically added by the IDE, I'll see if I can remove this later when this pr is done